### ISSUE TITLE
Masterlist update fails if TEMP on different partition than AppData (Windows)

### ISSUE THREAD

**Issue by spatulasnout** [loot/loot#1057]
It appears a `rename` is used to move the cloned masterlist repo to its final location, but this is failing (on Windows) if the `TMP` folder resides on a different partition than the user's AppData folder:
```
[12:55:19.539444] [warning]: No masterlist present at C:\Users\somebody\AppData\Local\LOOT\Oblivion\masterlist.yaml
[12:55:56.884580] [error]: Exception while executing query: boost::filesystem::rename: Access is denied: "S:\temp\LOOT-Oblivion-6a96-2404-cd2c-b209\.git", "C:\Users\somebody\AppData\Local\LOOT\Oblivion\.git"
```
Was able to work around this locally, by wrapping LOOT with a small `LOOT.bat` file, which changes the environment to place the temp folder location on the same partition as AppData:
```
@echo off
set TEMP=%LOCALAPPDATA%\LOOT-temp
set TMP=%LOCALAPPDATA%\LOOT-temp
.\LOOT.exe
```
(After having manually created the `%LOCALAPPDATA%\LOOT-temp` folder)

**Comment by Ortham** [comment]
Is this still a problem with [this snapshot build](https://bintray.com/loot/snapshots/download_file?file_path=loot_0.13.6-28-gc5917a5_update-api.7z)? (It requires the MSVC 2017 x86 redistributable.)

**Comment by spatulasnout** [comment]
Hi, I tried out the snapshot build, but I think it didn't get a chance to get to the rename/copy part of the code.
```
[21:50:27.311436] [error]: Exception while executing query: Git operation failed. Details: -3; revspec 'origin/v0.14' not found: libgit2 error
```
(I guess it's looking for a nonexistent branch or tag in the repo?)

**Comment by CptMcSplody** [comment]
@spatulasnout can you open the LOOT settings and change any showing as `v0.14` to `v0.13`

**Comment by spatulasnout** [comment]
Thanks! LOOT was now able to successfully update the masterlist using the above snapshot build, despite `TMP` being on a different partition than `AppData`. Appears to be fixed!

**Comment by Ortham** [comment]
Fixed in ecbe050ad699fbde5ab13f7614bd80feea2ee182.
